### PR TITLE
applications: asset_tracket_v2: Correct config for tests

### PR DESCRIPTION
**Pull request by dilner**
Adding CONFIG_LINKER_ORPHAN_SECTION_PLACE to all platforms.

Signed-off-by: David Dilner <david.dilner@nordicsemi.no>

### CODE REVIEW

**Review by simensrostad — APPROVED** [review]

**Review by coderbyheart — CHANGES_REQUESTED** [review]
Please explain in the commit message *why* CONFIG_LINKER_ORPHAN_SECTION_PLACE is added.

File: applications/asset_tracker_v2/tests/debug_module/prj.conf [hunk 1]
```diff
@@ -9,3 +9,4 @@ CONFIG_HEAP_MEM_POOL_SIZE=1024
 
 # Make CONFIG_EVENT_MANAGER_MAX_EVENT_CNT defined
 CONFIG_EVENT_MANAGER=y
+CONFIG_LINKER_ORPHAN_SECTION_PLACE=y
```
Comment (coderbyheart) [inline applications/asset_tracker_v2/tests/debug_module/prj.conf:12]:
```suggestion
# The following option is required to silence the linker warning about orphan
# sections. The event manager header files uses orphan sections.
CONFIG_LINKER_ORPHAN_SECTION_PLACE=y
```

File: applications/asset_tracker_v2/tests/gps_module/prj.conf [hunk 2]
```diff
@@ -16,3 +16,4 @@ CONFIG_NRF_MODEM_LIB=n
 
 # Make CONFIG_EVENT_MANAGER_MAX_EVENT_CNT defined
 CONFIG_EVENT_MANAGER=y
+CONFIG_LINKER_ORPHAN_SECTION_PLACE=y
```
Comment (coderbyheart) [inline applications/asset_tracker_v2/tests/gps_module/prj.conf:19]:
```suggestion
# The following option is required to silence the linker warning about orphan
# sections. The event manager header files uses orphan sections.
CONFIG_LINKER_ORPHAN_SECTION_PLACE=y
```

**Review by coderbyheart — CHANGES_REQUESTED** [review]
Alternative, keep the comment.

**Pull request closed without merging**